src/Encamina.Enmarcha.SemanticKernel.Connectors.Document/Utils/ImageHelper.cs

```diff
@@ -0,0 +1,34 @@
+﻿using CommunityToolkit.Diagnostics;
+
+using SixLabors.ImageSharp;
+
+namespace Encamina.Enmarcha.SemanticKernel.Connectors.Document.Utils;
+
+/// <summary>
+/// Provides helper methods for working with images.
+/// </summary>
+public static class ImageHelper
+{
+    /// <summary>
+    /// Gets the MIME type of an image from a stream.
+    /// </summary>
+    /// <param name="stream">The stream containing the image.</param>
+    /// <returns>The MIME type of the image, or "application/octet-stream" if the type is unknown.</returns>
+    /// <exception cref="ArgumentNullException">Thrown when the stream is null.</exception>
+    /// <exception cref="ArgumentException">Thrown when the stream does not contain a valid image.</exception>
+    public static string GetMimeType(Stream stream)
+    {
+        Guard.IsNotNull(stream);
+
+        try
+        {
+            var image = Image.Load(stream);
+
+            return image.Metadata.DecodedImageFormat?.DefaultMimeType ?? System.Net.Mime.MediaTypeNames.Application.Octet;
+        }
+        catch (Exception ex)
+        {
+            throw new ArgumentException("The stream does not contain a valid image.", ex);
```

Comment:
Unless there's a specific reason for doing so, I don't see the need to catch the exception here just to rethrow it as an `ArgumentException`. Additionally, the message "The stream does not contain a valid image." might not accurately reflect the actual cause of the exception. It might be better to remove the try/catch block and allow the original exception to be thrown.

Reply:
you are right, changed!